src/org/datafoodconsortium/connector/codegen/ruby/static/spec/importer_spec.rb

```diff
@@ -0,0 +1,156 @@
+describe DataFoodConsortium::Connector::Importer do
+  let(:enterprise) do
+    DataFoodConsortium::Connector::Enterprise.new(
+      "https://example.net/foo-food-inc",
+      suppliedProducts: [product, second_product],
+    )
+  end
+  let(:catalog_item) do
+    DataFoodConsortium::Connector::CatalogItem.new(
+      "https://example.net/tomatoItem",
+      product: product,
+    )
+  end
+  let(:product) do
+    DataFoodConsortium::Connector::SuppliedProduct.new(
+      "https://example.net/tomato",
+      name: "Tomato",
+      description: "Awesome tomato",
+      totalTheoreticalStock: 3,
+    )
+  end
+  let(:product_data) do
+    <<~JSON
+      {
+        "@context":"http://static.datafoodconsortium.org/ontologies/context.json",
+        "@id":"https://example.net/tomato",
+        "@type":"dfc-b:SuppliedProduct",
+        "dfc-b:name":"Tomato",
+        "dfc-b:description":"Awesome tomato",
+        "dfc-b:alcoholPercentage":0.0,
+        "dfc-b:lifetime":"",
+        "dfc-b:usageOrStorageCondition":"",
+        "dfc-b:totalTheoreticalStock":3
+      }
+    JSON
+  end
+  let(:product_data_with_context_v1_8) do
+    <<~JSON
+      {
+        "@context": {
+          "dfc-b": "https://github.com/datafoodconsortium/ontology/releases/latest/download/DFC_BusinessOntology.owl#",
+          "dfc-m": "https://github.com/datafoodconsortium/taxonomies/releases/latest/download/measures.rdf#",
+          "dfc-pt": "https://github.com/datafoodconsortium/taxonomies/releases/latest/download/productTypes.rdf#",
+          "dfc-b:hasType":{"@type":"@id"}
+        },
+        "@id":"https://example.net/tomato",
+        "@type":"dfc-b:SuppliedProduct",
+        "dfc-b:name":"Tomato",
+        "dfc-b:description":"Awesome tomato",
+        "dfc-b:alcoholPercentage":0.0,
+        "dfc-b:lifetime":"",
+        "dfc-b:usageOrStorageCondition":"",
+        "dfc-b:totalTheoreticalStock":3
+      }
+    JSON
+  end
+  let(:second_product) do
+    DataFoodConsortium::Connector::SuppliedProduct.new(
+      "https://example.net/ocra",
+      name: "Ocra",
+    )
+  end
+  let(:quantity) do
+    DataFoodConsortium::Connector::QuantitativeValue.new(
+      unit: piece,
+      value: 5,
+    )
+  end
+  let(:piece) do
+    unless connector.MEASURES.respond_to?(:UNIT)
+      #connector.loadMeasures(read_file("measures"))
```

Comment:
I don't think this should be commented out

Reply:
Ah, thank you. This was outdated code which I now removed. All vocabularies are loaded automatically in specs now.